script.js

```diff
@@ -4,12 +4,47 @@ var game = {
   deck: [],
   players: [],
   hands: [],
+
+  //Populates the `deck` array with 52 standard playing cards.
   buildDeck: function(){
+    for (var i = 0; i < suits.length; i++){
+      for (var m = 0; m < values.length; m++){
+        newCard = {
+          suit: suits[i],
+          value: values[m]
+        }
+        game.deck.push[newCard];
+        console.log(newCard);
+        console.log(newCard.length);
+      }
+    }
+  }, //end of builddeck
 
-  },
+  //Randomizes the order of `deck`.
   shuffleDeck: function(){
+    function shuffle(array) {
+      var currentIndex = array.length;
+      var temporaryValue;
+      var randomIndex;
 
-  },
+      // While there remain elements to shuffle...
+      while (0 !== currentIndex) {
```

Comment:
could also use a for loop, because you know how many times to loop `game.deck.length`